pkgs/development/ocaml-modules/ocaml-syntax-shims/default.nix

```
@@ -11,6 +9,8 @@ buildDunePackage rec {
     sha256 = "1j7848khli4p7j8i2kmnvhdnhcwhy3zgdpf5ds5ic30ax69y3cl9";
   };
 
+  minimalOCamlVersion = "4.02.3";
+
   useDune2 = true;
```

```suggestion
  duneVersion = "2";
```
This can also likely be 3